### DISCUSSION TITLE
Why people run games in SBIE?

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#2324] [General]
I can't think of **any** reason at all

**Comment by DavidXanatos** [comment]
> I can't think of **any** reason at all

There are quite a few reasons to do so:

1. to try out a mod (or update) and not mess up the existing installation safely
2. to actually find out where the game is storing save files. or to easily add more save options when the game originally supports only a limited amount of saves.
3. I was tolled by many people that they run multiple instances of some boring game that annoys their users with boring repetitive work to force them to buy official cheats.
4. easily move a game from one PC to an other, including save games, states and everything.

like with any other software.

**Reply by ghost** [reply]
1. To mod the game or install update, they would need to install the game in SBIE first.

I think they use SBIE to test if cracks work.

I don't think they get decent performance in such scenario anyway

**Reply by Domochevsky** [reply]
Personally speaking, I install all of my games into their individual sandboxes for compartmentalization reasons. So when I am done with them I can cleanly delete them with their box, leaving little to no traces on the rest of the system. No registry entries, no forgotten config files, no other oddities that modern games have a habit of shotgunning all over your system.

As a bonus, games I have not played in a while but plan on returning to, I can move completely to my backup harddrive. Easy one-drag move. Restoring them is the same in reverse. No wondering where it put its files or whether I got them all.

**Reply by ghost** [reply]
You can backup most games with few clicks. Same with games saves. Unless you play some really old games I guess you get low performance. Also, old games may introduce even more compatibility issues

**Reply by Domochevsky** [reply]
"But you can do these things in more roundabout ways" is not a good argument, I find. Having to search all over the system to see if I missed anything is not how I want to spend my time. Having it all in a single box makes for consistent behavior. :)

That said, I'm getting this vibe from you that you are less interested in reasons and more in things to argue with. Correct me if I'm wrong. Whazzat about?

**Reply by ghost** [reply]
SBIE was made as security software, not as software virtualization layer. I don't have to check to know that games which require communication with services will not work. None of you mention any game name. If those games are so simple, you could just zip them to make them portable and there is a chance it would be enough.

I don't argue, just curious.

Another reason is to run multiple bots...
